internal/api_rest.go

```diff
@@ -13,17 +13,14 @@ func Router() {
 		router.Route("/comedores", func(r chi.Router) {
```

Comment:
1. Estás usando `r` como identificador. ¿Usar identificadores de una sola letra es una buena práctica en código limpio *o* en Go?
2. Estás usando *el mismo* identificador para todas las funciones anidadas. ¿Crees que eso es una buena práctica?
3. Igualmente, ¿crees que meter todas las funciones inline contribuye a la legibilidad del código? ¿O a la "separation of concerns"?

Reply:
Los puntos 1 y 2 quedan solucionados en [8fc4534](https://github.com/antoniorr02/MenuConsulter/pull/46/commits/8fc45343741abe53e0017caf945ac42a0ef2cd9e)

Por otro lado, el punto 3, queda solucionado en [5baafc2](https://github.com/antoniorr02/MenuConsulter/pull/46/commits/5baafc28e603e427b88bd6c1d84dfbfb5a945506)
De forma que las rutas diseñadas resultantes mantendrían la siguiente lógica:
La jerarquía de rutas en tu código queda definida con la siguiente forma cumpliendo con mis HUs:
- Obtener todos los comedores: /comedores
- Comedor específico: /comedores/{nombre_comedor}
- Menús dentro de un comedor: /comedores/{nombre_comedor}/menus
- Menú específico: /comedores/{nombre_comedor}/menus/{fecha}
- Platos dentro de un menú: /comedores/{nombre_comedor}/menus/{fecha}/platos
- Plato específico: /comedores/{nombre_comedor}/menus/{fecha}/platos/{nombre_plato}